packages/grid/x-data-grid-premium/src/hooks/features/aggregation/gridAggregationFunctions.ts

```diff
@@ -17,14 +17,23 @@ const sumAgg: GridAggregationFunction<unknown, number> = {
   columnTypes: ['number'],
 };
 
-const avgAgg: GridAggregationFunction<number> = {
-  apply: (params) => {
-    if (params.values.length === 0) {
+const avgAgg: GridAggregationFunction<unknown, number> = {
```

Comment:
Since the output for this function will change for the same data now, it may break unit tests or Argos tests for some users, should we announce this somehow?

Reply:
Could be, but it's a bugfix, and if their test breaks - it means it has to be updated anyway to match the proper behavior 🤷‍♂️